src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -127,25 +127,51 @@ <h4 *ngIf="isImport" id="jhi-programming-exercise-heading-import" jhiTranslate="
-                    />
-                    <ng-container *ngFor="let e of maxScore.errors! | keyvalue | removekeys: ['required']">
-                        <div *ngIf="maxScore.invalid && (maxScore.dirty || maxScore.touched)" class="alert alert-danger">
-                            <div [jhiTranslate]="'artemisApp.exercise.form.maxScore' + '.' + e.key"></div>
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group-narrow">
+                            <label class="label-narrow" jhiTranslate="artemisApp.exercise.points" for="field_points">Regular Points</label>
+                            <input
+                                required
+                                type="number"
+                                [pattern]="maxScorePattern"
+                                class="form-control"
+                                min="1"
+                                max="9999"
+                                name="points"
+                                id="field_points"
+                                [(ngModel)]="programmingExercise.maxScore"
+                                #maxScore="ngModel"
+                            />
+                            <ng-container *ngFor="let e of maxScore.errors! | keyvalue | removekeys: ['required']">
+                                <div *ngIf="maxScore.invalid && (maxScore.dirty || maxScore.touched)" class="alert alert-danger">
+                                    <div [jhiTranslate]="'artemisApp.exercise.form.maxScore' + '.' + e.key"></div>
+                                </div>
+                            </ng-container>
                         </div>
-                    </ng-container>
+                    </div>
+                    <div class="col">
+                        <div class="form-group-narrow">
+                            <label class="label-narrow" jhiTranslate="artemisApp.exercise.bonusPoints" for="field_bonusPoints">Bonus Points</label>
+                            <input
+                                required
+                                type="number"
+                                [pattern]="maxScorePattern"
+                                class="form-control"
+                                min="1"
+                                max="9999"
+                                name="bonusPoints"
+                                id="field_bonusPoints"
+                                [(ngModel)]="programmingExercise.bonusPoints"
+                                #maxScore="ngModel"
+                            />
+                            <ng-container *ngFor="let e of maxScore.errors! | keyvalue | removekeys: ['required']">
+                                <div *ngIf="maxScore.invalid && (maxScore.dirty || maxScore.touched)" class="alert alert-danger">
+                                    <div [jhiTranslate]="'artemisApp.exercise.form.points' + '.' + e.key"></div>
```

Comment:
This needs to be bonusPoints.
```suggestion
                                    <div [jhiTranslate]="'artemisApp.exercise.form.bonusPoints' + '.' + e.key"></div>
```